sql/core/src/main/scala/org/apache/spark/sql/SparkSessionExtensions.scala

```diff
@@ -199,6 +199,21 @@ class SparkSessionExtensions {
     optimizerRules += builder
   }
 
```

Comment:
Could you update the description above, too?
https://github.com/apache/spark/blob/29096a8869c95221dc75ce7fd3d098680bef4f55/sql/core/src/main/scala/org/apache/spark/sql/SparkSessionExtensions.scala#L37-L48

Reply:
Done, thanks for cathing this!